### PR TITLE
Add documentation for contributing to Aesara

### PR DESCRIPTION
**Pull request by dgerlanc**
Add a "Contributing" section in the `README.rst` and update `CONTRIBUTING.md`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+[Numba](https://github.com/numba/numba).
+
+We welcome patches with bug fixes, and we’re happy to answer questions if you’ve
+already put in some effort to find the answer yourself. Please note, however,
+that we’re _unlikely_ to consider new feature contributions or design changes
+unless there’s a strong argument that they are fully in line with our stated
+goals. If you’re not sure, just ask.
+
+Our technical scope and project governance may open up later, of course,
+especially after we release version 1.0. For now, though, we would characterize
```
Comment (brandonwillard) [inline CONTRIBUTING.md:23]:
```suggestion
Our technical scope and project governance may open up later, of course. For now, though, we would characterize
```

**Review by brandonwillard — DISMISSED** [review]
Just a comment about the version; otherwise, looks great.

**Review by rlouf — COMMENTED** [review]
Looks great! Two comments on the content:

- How about removing the section about moving PRs from Theano? I think this is neither necessary, nor desirable at this point.
- We should add a comment about what we think issues are about (stuff that someone can pick up and work on immediately) and when to open a discussion instead. Insist on MWE and the fact that they need to be implemented with _Aesara_ directly.

**Review by brandonwillard — APPROVED** [review]
Looks great!

**Pull request merged**